stable/elasticsearch/templates/client-svc.yaml

```diff
@@ -17,6 +17,9 @@ spec:
   ports:
     - name: http
       port: 9200
+{{- if and .Values.client.nodePort (eq .Values.client.serviceType "NodePort") }}
```

Comment:
Maybe we should be more precise as there can be several ports in the future, like `client.httpNodePort` ?

Reply:
This change makes sense! :) Thanks for the feedback!